scripts/helpers.sh

```diff
@@ -10,7 +10,8 @@ alias klaxon-rails="klaxon-app rails"
 # Shorter way to run the Ruby rake command.
 alias klaxon-rake="klaxon-app rake"
 
-# TODO: add command to run tests
+# Run tests
+alias klaxon-test="klaxon-app bundle exec rspec"
```

Comment:
Unrelated to this PR, but I wanted to run tests (one fails), and I thought I should add a helper to do it